lib/internal/main/test_runner.js

```diff
@@ -1,18 +1,25 @@
-#!/usr/bin/env node
-// https://github.com/nodejs/node/blob/484d35402de36d8d5756b244c8e5fbb8aa4c6afd/lib/internal/main/test_runner.js
+// https://github.com/nodejs/node/blob/1aab13cad9c800f4121c1d35b554b78c1b17bdbd/lib/internal/main/test_runner.js
 'use strict'
 const {
   ArrayFrom,
+  ArrayPrototypeFilter,
+  ArrayPrototypeIncludes,
   ArrayPrototypePush,
   ArrayPrototypeSlice,
   ArrayPrototypeSort,
   Promise,
   SafeSet
 } = require('#internal/per_context/primordials')
+// const {
+//   prepareMainThreadExecution
+// } = require('#internal/bootstrap/pre_execution')
```

Comment:
would it be simpler to mock this file but have it noop? Same for `console` below